### PR TITLE
PXS-478 Add basic app template

### PR DESCRIPTION
**Pull request by timotheeduran**
This PR adds the boilerplate code for an empty android app displaying _hello world_. This also includes a basic CI.

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+My Application
```
Comment (xavierpantet) [inline .idea/.name:1]: Can we have a better name here?
Like maybe Pindex Android or Pindex Android App?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">My Application</string>
```
Comment (xavierpantet) [inline app/src/main/res/values/strings.xml:2]: Can we have a better name here?
Like Pindex Android or Pindex Android App?

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "My Application"
```
Comment (xavierpantet) [inline settings.gradle:2]: Can we have a better name here?
Like Pindex Android or Pindex Android App?

**Review by xavierpantet — CHANGES_REQUESTED** [review]

File: .idea/.name (hunk 1 above)
Comment (timotheeduran) [reply .idea/.name:1]: No, this is a basic template. This is done in PR 477

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (timotheeduran) [reply app/src/main/res/values/strings.xml:2]: No, this is a basic template. This is done in PR 477

File: settings.gradle (hunk 3 above)
Comment (timotheeduran) [reply settings.gradle:2]: No, this is a basic template. This is done in PR 477

**Review by xavierpantet — APPROVED** [review]

**Pull request merged**